cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java

```diff
@@ -47,9 +51,15 @@ public static final class ScheduledJob implements Job {
 
     private static final Logger LOG = LoggerFactory.getLogger(ScheduledJob.class);
     private final ScheduleTaskPublisher taskPublisher;
+    private final MetricsCollectionService metricsCollectionService;
+    private static final Map<String, String> METRICS_CONTEXT = ImmutableMap.of(
```

Comment:
Can we add the application_name and schedule_name as well in the context? Wondering, otherwise how we will know the metric is emitted for which pipeline?
@janakigg WDYT?

Reply:
I can see pipeline id in successful pipeline runs. Successful scheduling metric is after launch which has some dependency on the pipeline. So it might make sense to add these to the metric.

Reply:
added application and schedule to the context